test/e2e/framework/volume/fixtures.go

```diff
@@ -488,6 +489,18 @@ func runVolumeTesterPod(client clientset.Interface, config TestConfig, podSuffix
 
 	for i, test := range tests {
 		volumeName := fmt.Sprintf("%s-%s-%d", config.Prefix, "volume", i)
+
+		// We make the container privileged when SELinux is enabled on the host,
+		// so the test can write data to a location like /tmp. Also, due to the
+		// Docker bug below, it's not currently possible to map a device with
+		// a privileged container, so we don't go privileged for block volumes.
+		// https://github.com/moby/moby/issues/35991
+		privileged := false
+		if selinux.SELinuxEnabled() && test.Mode != v1.PersistentVolumeBlock {
```

Comment:
This needs to be replaced by something else because it doesn't detect if SELinux is enabled on the host

Reply:
@bertinatto could you please create an issue for this action item ?

Reply:
Sure, I'll create an issue as soon as I figure this out